packages/server/modules/comments/tests/comments.graph.spec.js

```diff
@@ -978,8 +978,8 @@ describe('Graphql @comments', () => {
             [archiveOthersComment, false],
             [editOthersComment, false],
             [replyToAComment, false],
-            [queryComment, false],
-            [queryComments, false],
+            [queryComment, true],
```

Comment:
querying on public projects should be possible even for archived users, that's the same logic we have for Project.canRead